apps/cyberstorm-storybook/LinkLibrary.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+  LinkLibrary,
+  ThunderstoreLinkProps,
+} from "@thunderstore/cyberstorm/src/components/LinkingProvider";
+import React from "react";
+
+interface CyberstormLinkProps {
+  children?: React.ReactNode;
+}
+
+interface LinkProps extends CyberstormLinkProps, ThunderstoreLinkProps {
```

Comment:
As far as I can tell, we do nothing with ThunderstoreLinkProps here. If that's so, it should perhaps be yeeted.